pages/linux/i3.md

```diff
@@ -0,0 +1,40 @@
+# i3
+
+> A dynamic tiling window manager.
+> More information:<https://i3wm.org/docs/userguide.html>.
+
+- Start i3 (Note you must not have a window manager open when using this command.):
+
+`i3`
+
+- Open a new terminal window:
+
+`Super+Return`
```

Comment:
Maybe we could have spaces either side of the `+` here? I think it would make it easier to read at a glance, and we do this on other pages if I recall correctly.

Reply:
Ah didn't notice that my bad. I'll edit the rest of the document to follow that!